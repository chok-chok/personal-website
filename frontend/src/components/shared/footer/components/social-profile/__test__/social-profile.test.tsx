import * as React from 'react';
import SocialProfile from '../social-profile';
import { shallow } from 'enzyme';

describe('Test SocialProfile component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<SocialProfile />);
    expect(tree).toMatchSnapshot();
  });
});
