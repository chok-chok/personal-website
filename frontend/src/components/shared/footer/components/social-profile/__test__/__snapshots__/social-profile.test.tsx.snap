// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SocialProfile component should renders correctly 1`] = `
<styled__StyledSocialProfileBackground>
  Social Profile
</styled__StyledSocialProfileBackground>
`;
